generated/xamarin-flexible-sync/realm-todo-dotnet/TaskPage.xaml.cs

```diff
@@ -53,9 +55,7 @@ private void SetUpTaskList()
         {
             if (_tasks == null || _tasks.Count() == 0)
```

Comment:
If `_tasks==null` it makes sense to initialize it, but if `Count() == 0`, then it doesn't. If there are no tasks, then there are not tasks, assigning it again will not change it